content/blog/2020/06/2020-06-12-infra-and-aws-donation.adoc

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "AWS and Infrastructure Updates - Hello from Infra"
```

Comment:
I do not get "Hello from Infra" TBH. Maybe just "Jenkins infrastructure sponsorship by AWS"?

Reply:
Imho it's not only about AWS but more global hightlight on sponsoring and contribution

Reply:
Personally I would just title it "Infrastructure News"

Reply:
```suggestion
title: "Jenkins Infrastructure: Stats, Updates, and AWS sponsorship"
```

or so?

Reply:
+1 I like the updated title suggested here.